packages/cypress/test/package.json

```diff
@@ -5,6 +5,7 @@
         "express": "4.17.1",
         "cypress": "^5.2.0",
         "@types/node": "14.0.14",
-        "typescript": "3.9.5"
+        "typescript": "3.9.5",
+        "tar": "6.0.5"
```

Comment:
would be nice to avoid an extra dependency here too

Reply:
This dependency is necessary for `//packages/cypress/test` due to a convoluted test setup procedure (since `@bazel/cypress` isn't actually installed we still need its dep on tar to be listed somewhere)

Reply:
yup that makes sense